test/mbedls_toolsbase.py

```diff
@@ -351,6 +351,7 @@ def test_fs_before(self):
             'mount_point': 'invalid_mount_point',
             'serial_port': 'invalid_serial_port'
         }
+        self.base.list_unmounted = True
```

Comment:
Pretty sure I'm not getting this cuz my brain is dead, but why is this needed?

Reply:
Cuz the test mount points don't exist.

Reply:
Summing up what we discussed offline: we should be mocking anytime the filesystem would be accessed. So in this case we need to mock the correct `find_candidates` call.